packages/cli/src/swc/sources.ts

```diff
@@ -106,10 +106,20 @@ export async function requireChokidar() {
     }
 }
 
-export async function watchSources(sources: string[], includeDotfiles = false) {
+export async function watchSources(
+    sources: string[],
+    includeDotfiles = false,
+    only: string[] = [],
+    ignore: string[] = [],
+) {
     const chokidar = await requireChokidar();
-
-    return chokidar.watch(sources, {
+    const sourceFiles = await globSources(
```

Comment:
What happens if a file is created after this call?

Reply:
Good question. Should be the same as what happened before with their internal globbing and their documented v4 solution, new files don’t get observed after the glob is expanded. I can play around with a few scenarios this week to validate.